script.js

```diff
@@ -0,0 +1,27 @@
+// one-var
+
+var a = 10;
+
+const value = 10;
+value++;
+
+//comment whitout space
+
+const func = function(){
+
+  let ad = 10
+  var q=45
+
+}
+
+
+const funcName = ()=>{
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Block must be padded by blank lines.](https://app.codacy.com/gh/WebHeroSchool/js_test2/pullRequest?prid=5502592)